images/macos/provision/core/openssl.sh

```diff
@@ -2,9 +2,6 @@
 
 source ~/utils/utils.sh
 
-echo "Installing OpenSSL..."
-export PATH="/usr/local/bin:/usr/local/sbin:~/bin:$PATH"
```

Comment:
Do you know what `ln -sf $(brew --prefix openssl)/bin/openssl /usr/local/bin/openssl` does? Do we need it on Mojave and HS after removing openssl 1.0.2?

cc: @al-cheb for any context

Reply:
Start with macOS High Sierra is LibreSSL default one. The  `brew install openssl`  command ,by default, is not symlinked into /usr/local, that's why we should create an additional symlink.

Reply:
Done